src/App.tsx

```diff
@@ -1,61 +1,122 @@
+import { useState } from 'react';
 import './App.scss';
 
-// import usersFromServer from './api/users';
-// import todosFromServer from './api/todos';
+import usersFromServer from './api/users';
+import todosFromServer from './api/todos';
+import { TodoList } from './components/TodoList/TodoList';
+import { User } from './components/types/User';
+import { Todo } from './components/types/Todo';
+
+function getUser(userId: number): User | null {
+  const foundUser = usersFromServer.find(user => user.id === userId);
+
+  return foundUser || null;
+}
+
+const todos: Todo[] = todosFromServer.map(todo => ({
+  ...todo,
+  user: getUser(todo.userId),
+}));
+
+export const App: React.FC = () => {
+  const [title, setTitle] = useState('');
+  const [userId, setUserId] = useState(0);
+  const [titleError, setTitleError] = useState(false);
+  const [userIdError, setUserIdError] = useState(false);
+
+  const handleSumbit = (event:React.FormEvent<HTMLFormElement>) => {
+    event.preventDefault();
+
+    if (userId === 0 || title === ' ') {
```

Comment:
- the `0` looks kinda hardcoded. I'd say that it is better to store null there and to check whether the userId is `truthy`
- what is `title === ' '`? To me it seems to be very similar to the `trim()` method